src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -89,6 +100,91 @@ const TeamsTools = () => {
     </TableRow>
   ));
 
+  const handleSaveDesc = async () => {
+    if (!isUnderWordCount(currentDescription, 300)) {
+      return alert('Description must be less than 300 words');
+    }
+    resetDefaults();
+    setPrevDescription(currentDescription);
+  };
+
+  const resetDefaults = () => {
+    setIsEditingDescription(false);
+    setIsPreview(false);
+  };
+
+  const hasPerms = auth.user && isAdmin(auth.user.role);
+
+  interface DescriptionButtonsProps {
+    toRight: ReactNode;
+  }
+
+  const DescriptionButtons: React.FC<DescriptionButtonsProps> = ({ toRight }) => {
+    return (
+      <Box
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          marginTop: '6px',
+          marginBottom: '10px'
+        }}
+      >
+        <FormLabel>Description</FormLabel>
+        {toRight}
+      </Box>
+    );
+  };
+
+  const editButtons = (
+    <Box style={{ display: 'flex' }}>
+      <Button
+        onClick={() => {
+          setCurrentDescription(prevDescription);
+          resetDefaults();
+        }}
+      >
+        Cancel
+      </Button>
+      <Button
+        onClick={() => setIsPreview(!isPreview)}
+        sx={{
+          ml: 2,
+          backgroundColor: theme.palette.grey[600],
+          color: theme.palette.getContrastText(theme.palette.grey[600]),
+          '&:hover': {
+            backgroundColor: theme.palette.grey[700]
+          }
+        }}
+      >
+        {isPreview ? 'Edit' : 'Preview'}
+      </Button>
+
+      <Button
```

Comment:
Seems like the save button is uneccessary? The description saves even if you never click it, at least when I test it